vagrant/provision-master.sh

```diff
@@ -37,13 +37,15 @@ systemctl start docker.service
 # Create systemd service
 cat <<EOF > /usr/lib/systemd/system/openshift-master.service
 [Unit]
-Description=openshift master
-Requires=docker.service
+Description=OpenShift Master
+Requires=docker.service network.service
 After=network.service
 
 [Service]
 ExecStart=/usr/bin/openshift start master --public-master=${MASTER_IP} --nodes=${node_list}
 WorkingDirectory=/vagrant/
+Restart=on-failure
+RestartSec=10s
```

Comment:
Yes, my take would be to remove it. I think we want this to crash and burn if its not good :).